config/routes.rb

```diff
@@ -11,9 +11,17 @@
   post '/tasks' => 'task#create'
 
   get '/tasks/:id' => 'task#show'
+  get '/tasks/:id/edit' => 'task#edit'
+  patch '/tasks/:id/edit' => 'task#update'
 
+
+  patch '/tasks/:id' => 'task#update_status'
   delete '/tasks/:id' => 'task#destroy'
 
+  get '/people' => 'people#index'
+  get 'people/:id' => 'people#show'
```

Comment:
I think you'd want to be consistent with your specific routes and whether or not you'd include the `/` at the start